packages/storybook/stories/CallWithChatComposite/CallWithChatCompositeDocs.tsx

```diff
@@ -12,6 +12,21 @@ const formFactorSnippet = `
 <CallWithChatComposite formFactor="mobile" />
 `;
 
+const addFileSharingSnippet = `
+<CallWithChatComposite
+adapter={chatAdapter}
+options={{
+  fileSharing: {
+    uploadHandler: fileUploadHandler,
+    // If fileDownloadHandler is not provided. The file URL is opened in a new tab.
+    // You can find examples of fileDownloadHandler and fileUploadHandler in the tutorial linked above.
```

Comment:
Are we providing the link to tutorial in storybook as mentioned in the comment

Reply:
Its in the snippet just above.  I can put a link in the comment.